cpp/oneapi/dal/detail/cpu.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright 2020 Intel Corporation
+* Copyright 2024 Intel Corporation
```

Comment:
is it necessary to change Copyright here? +please, use the latest copyright rule